Toggl.Foundation.MvvmCross/ViewModels/StartTimeEntryViewModel.cs

```
@@ -286,6 +294,43 @@ public override void Prepare(StartTimeEntryParameters parameter)
             hasAnyProjects = (await dataSource.Projects.GetAll()).Any();
         }
 
+        protected override void SaveStateToBundle(IMvxBundle bundle)
+        {
+            try {
+                var textFieldInfo = JsonConvert.SerializeObject(TextFieldInfo);
+                bundle.Data[StateBundleParams.TextFieldInfo.ToString()] = textFieldInfo;
+            } 
+            catch (JsonSerializationException) { }
```

If this fails, the exception is swallowed. That means the state you save to the bundle will not be complete and that reload from bundle won't be complete either, leading to inconsistent state of the app.